crates/next-custom-transforms/tests/fixture/next-font-with-directive/use-cache/output.js

```diff
@@ -6,8 +6,9 @@ import inter from '@next/font/google/target.css?{"path":"app/test.tsx","import":
 export var $$RSC_SERVER_CACHE_0 = $$cache__("default", "c0dd5bb6fef67f5ab84327f5164ac2c3111a159337", 0, async function Cached({ children }) {
     return <div className={inter.className}>{children}</div>;
 });
+registerServerReference($$RSC_SERVER_CACHE_0, "c0dd5bb6fef67f5ab84327f5164ac2c3111a159337", null);
 Object["defineProperty"]($$RSC_SERVER_CACHE_0, "name", {
     value: "Cached",
     writable: false
 });
-export var Cached = registerServerReference($$RSC_SERVER_CACHE_0, "c0dd5bb6fef67f5ab84327f5164ac2c3111a159337", null);
+export var Cached = $$RSC_SERVER_CACHE_0;
```

Comment:
One might wonder why we aren't just using the original export name as the action/cache name, instead of generating the additional one. The reason is that the `$$RSC_SERVER_` prefixed exports get a special treatment during tree shaking.